.github/workflows/update-version.yml

```diff
@@ -0,0 +1,37 @@
+name: Update Version
+on:
+  workflow_dispatch:
+    inputs:
+      version: 
+        description: 'New version to use for the Places extension. Example: 3.0.0'
+        required: true
+
+      core-dependency:
+        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
+        required: false
```

Comment:
Should it be `required: true` since Places has hard dependency on Core?

Reply:
Also let's remove `[Optional]`

Reply:
Agreed this field is not mandatory. Core version dependency can remain the same despite the update to Places version 
 Keeping them the same way as other extension
 https://github.com/adobe/aepsdk-userprofile-android/blob/main/.github/workflows/update-version.yml#L10-L12